packages/block-editor/src/style.scss

```diff
@@ -27,7 +27,7 @@
 @import "./components/duotone-control/style.scss";
 @import "./components/font-appearance-control/style.scss";
 @import "./components/global-styles/style.scss";
-@import "./components/grid-visualizer/style.scss";
+@import "./components/grid/style.scss";
```

Comment:
Just a quick drive-by comment: now that the `style.scss` has been moved back to `grid-visualizer`, looks like this line needs to be reverted, too?

Reply:
oh well spotted, thanks! updated.